packages/toolpad-components/src/Form.tsx

```diff
@@ -19,7 +16,7 @@ export const FormContext = React.createContext<{
 
 interface FormProps extends ContainerProps {
   value: FieldValues;
-  onChange: (newValue: FieldValues) => void;
+  onChange?: (newValue: FieldValues) => void;
```

Comment:
Why should this be optional?

Reply:
No reason, leftover I forgot to remove. Updated